.merlin

```diff
@@ -0,0 +1,23 @@
+PKG stdext
+PKG squeeze
```

Comment:
This should be removed because this ocamlfind package is provided by this repo itself (it's in the oasis file as a `Library`). merlin complained to me about missing this package, but when I removed it it worked fine, and I could still jump into this library - no definitions were missing. I think that's becaues merlin can find the build files of this library in the build dir, since it belongs to this repo.